@here/harp-datasource-protocol/lib/Techniques.ts

```diff
@@ -481,6 +481,10 @@ export interface TerrainTechnique extends MakeTechniqueAttrs<TerrainTechniquePar
     name: "terrain";
 }
 
+export interface LabelRejectionLineTechnique extends MakeTechniqueAttrs<BaseTechniqueParams> {
```

Comment:
What is `width` of this line ?
As i understand, now we check collisions only against abstract infinitely-thin lines.

Reply:
Width is 0. I.e. we just check if the line intersects the box, without any consideration of its width.